ocrd_utils/ocrd_utils/__init__.py

```diff
@@ -5,29 +5,48 @@
 
 The functions have the syntax X_from_Y, where X/Y can be
 
-points a string encoding a polygon: "0,0 100,0 100,100, 0,100"
-polygon an array of x-y-tuples of a polygon: [[0,0], [100,0], [100,100], [0,100]]
-xywh a dict with keys for x, y, width and height: {'x': 0, 'y': 0, 'w': 100, 'h': 100}
-points is for PAGE
+* bbox is a 4-tuple of x0, y0, x1, y1 of the bounding box
+* points a string encoding a polygon: "0,0 100,0 100,100, 0,100". points is for PAGE
+* polygon an array of x-y-tuples of a polygon: [[0,0], [100,0], [100,100], [0,100]]
+* xywh a dict with keys for x, y, width and height: {'x': 0, 'y': 0, 'w': 100, 'h': 100}
 
 polygon is what opencv2 expects
```

Comment:
Please add our own coordinate provisions (`coordinates_of_segment`, `coordinates_for_segment`, `polygon_mask`, `image_from_polygon`, `rotate_coordinates`) as use case for `polygon` here.
```suggestion
polygon is what opencv2 and higher-level coordinate functions in ocrd_utils expect
```